### DISCUSSION TITLE
unbalanced mod ship questions

### DISCUSSION THREAD

**Discussion by daggs1** [endless-sky/endless-sky#7693] [General]
Greetings,

I have a new modified ship that feels a bit unbalanced, I'd appreciate incites on it.
I have a Dreadnought with the following new outfits:
power: Boulder Reactor, 2 Millennium Cell and 4 Crystal Capacitor
shielding: Hai Diamond Regenerator
weapons: Ion Cannon, 2 Pulse Turret, 2 Plasma Cannon, 2 Point Defense Turret and a Plasma Turret
cooling: 2 Large Heat Shunt and a Small Heat Shunt

when in battle, it seems that:
 - I can fire has much as I want, I don't see any heating at all and the power bar is always at max.
 - it looks like the shield recharging takes longer than I has with my previous ship (aerie)
 - the firepower seems weaker than expected, I got into a battle head to head with a pirate leviathan and lost somehow.
 
so my questions these:
1. is it possible that I don't need all the capacitors? 2 or 1 are enough with the 2 power reactors?
2. what can affect the slow shield recharging?
3. what change can I do to the weapons to increase the power? maybe another pulse cannon? or switch to 2 Plasma Turret and one Pulse Turret?
4. do I need all that cooling? 2 large Shunts and one small?

Thanks.

p.s. small unrelated question, I scanned an author ship and saw it has both jump and hyper drives, I assume that there is logic in that, right?

**Comment by daggs1** [comment]
ran a test, removed all the shunts but large one, stayed with Crystal Capacitor, power bar got empty after ~60 seconds of constant fire and maneuvering, it looks that regardless of the bar's high, the firing stayed the same, when I stopped firing, it took less than 10 seconds to recharge the bar up to it's fullness.
the heat bar got eventually to 60% and stayed there.
I assume that weapons change affect the heating (I'll reinstate the small shunt) will deplete the power bar faster but with such quick recharge rate, I doubt it matter much.

**Comment by mOctave** [comment]
> 2. what can affect the slow shield recharging?

You might want a bigger shield generator. I believe Coalition tech is also more efficient for shield generation, so you could look into that.

> 3. what change can I do to the weapons to increase the power? maybe another pulse cannon? or switch to 2 Plasma Turret and one Pulse Turret?

If you are up to stealing Quarg or Pug technology, you could do that. Electron beam weapons are also pretty high damage, presuming you've unlocked them, although they're also pretty big.

> 4. do I need all that cooling? 2 large Shunts and one small?

Probably not, unless you're going into alien (read Remnant and Korath) space a lot, if there's no noticeable heating.

>  p.s. small unrelated question, I scanned an author ship and saw it has both jump and hyper drives, I assume that there is logic in that, right?

Yes, and no. It was probably added for fun by the author, but jump drives do use 2x the fuel of hyperdrives, so it sort of makes sense.

**Comment by bene-dictator** [comment]
If power is always at max, either you don't need all those crystal capacitators because it's full or the storage on those crystal capacitators are so much that, again, you could get away with removing a few of them. I am not sure of better generator options other than wanderer reactors. 

I always try to fit a systems core into my ship, but I wonder if they are newb traps, because they take up a lot of room. I might look at coalition shielding or wanderer shielding (if you can get it). In addition, coalition active cooling is really good cooling for their size, only that it has a power requirement that might become troublesome. Especially if you're able to use up all your power after 60 seconds. 

I like to stick a hyperdrive alongside a jump drive on my flagship because listening to the jump drive activation sound over and over again will give me tinnitus some day, but it's not that efficient when you could use that space for another weapon or some cargo space for missions that need them. 

I was never impressed with plasma turrets because plasma projectiles are really slow, and I don't really know that much about ion cannons' usefulness. They can be very annoying in Hai space, but otherwise they take up a lot of room. If you're losing to a Leviathan head on, you must have taken it on at a significant disadvantage, because Leviathans are worse than Dreadnoughts in every way I can think of. If you have better engines, you should be able to maneuver in a way that forces the enemy ship to turn while you still have your guns pointed at them. But engines draw a lot of electricity.

**Comment by daggs1** [comment]
thanks for the input guys, I'll try it out
